src/index.js

```diff
@@ -1,14 +1,125 @@
-import './index.css';
-import addHtml from './modules/caling.html.js';
-import btnAdd from './modules/callinf.buttonAdd.js';
-import refreshBtn from './modules/calling.reload.js';
-// eslint-disable-next-line camelcase
-import { clear_my_Funct } from './modules/calling.check.js';
-
-const buttonAdd = document.getElementById('add-button');
-buttonAdd.addEventListener('click', btnAdd);
-const refresh = document.getElementById('refresh');
-refresh.addEventListener('click', refreshBtn);
-const clear = document.getElementById('clear');
-clear.addEventListener('click', clear_my_Funct);
-addHtml();
+/* eslint-disable  import/no-cycle, import/no-mutable-exports */
```

Comment:
- [ ] Using linters can help developers write better code by enforcing consistent code styles, identifying common programming errors, and highlighting potential performance issues. Kindly remove the lines where linters are disabled. 🤝️ Please do not modify any of the linter configuration files when addressing the issue. 🥸️